send/generic_send

```diff
@@ -22,6 +22,19 @@ DESTINATION_TYPE_EMAIL="email"
 DESTINATION_TYPE_HOST="host"
 DESTINATION_TYPE_USER_HOST="user@host"
 DESTINATION_TYPE_USER_HOST_PORT="user@host:port"
+DESTINATION_TYPE_USER_HOST_WINDOWS="user@host-windows"
+DESTINATION_TYPE_HOST_WINDOWS_PROXY="host-windows-proxy"
+
+#This function runs "$FILTER" as a command.
+#If "$FILTER" equals "$DESTINATION",
+#the function runs base64 command and adds the "$DESTINATION" to the end of its output.
+function filter {
+	if [ "$FILTER" = "$DESTINATION" ]; then
```

Comment:
It's functional, but the code is not nice. 
I would prefer to store only constants to $FILTER (e.g. "none", "base64", "host-windows-proxy") and then modify the result of the filter function based on that.

Reply:
I have completely erased FILTER variable. The filter function now depends on DESTINATION_TYPE. It seemed to me like the best solution but if it is not ok I will change it the way you wanted.